mleap-tensorflow/README.md

```diff
@@ -1,34 +1,18 @@
 # Tensorflow
-
```

Comment:
there is a RELEASE_NOTES.md file in the repo root as well.  Can you add a line in there about the change.

Reply:
Sure.

Reply:
updated.